Specifications/Compiler/com.apple.compilers.llvm.clang.1_0.migrator.xcspec

```diff
@@ -0,0 +1,41 @@
+/**
+ Copyright (c) 2015-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+(
```

Comment:
Can you split this into two files? The specifications so far are one-per-file, which makes them much easier to maintain since the mapping is one to one and the filenames are the same as the identifiers.